.github/workflows/iriscast_package_build.yaml

```diff
@@ -34,3 +34,67 @@ jobs:
       - name: Analyse with pylint
         run: |
           cd iriscasttools && pylint --recursive yes test iriscasttools
+
+  build_wheel:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["3.x"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: "pip"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+
+      - name: Build Wheels
+        run: cd iriscasttools && python setup.py bdist_wheel
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: iriscasttools
+          path: iriscasttools/dist
+          if-no-files-found: error
```

Comment:
Do we need to tag these artefacts in any way? Such as the branch SHA or similar?